draft/2021-12-29-this-week-in-rust.md

```diff
@@ -23,6 +23,7 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 * [Nushell 0.42](https://www.nushell.sh/blog/2021-12-28-nushell_0_42.html)
 * [gfx-rs](https://gfx-rs.github.io/2021/12/25/this-year.html)
 * [BonsaiDb December Update: Finishing up alpha 1](https://community.khonsulabs.com/t/bonsaidb-december-update-finishing-up-alpha-1/88)
+* [This Year in Embedded Rust: 2021 Edition](https://blog.rust-embedded.org/this-year-in-embedded-rust-2021/)
```

Comment:
unclear if this should be under official, probably should be

Reply:
Generally we reserve Official for the formal Rust blog and Inside Rust blog. I this case....I'd like to keep it under project updates.